indymeet/settings/production.py

```diff
@@ -38,6 +38,7 @@
 except ImportError:
     pass
 
+BASE_URL = 'https://djangonaut.space'
```

Comment:
I'd suggest using `BASE_URL = os.getenv("BASE_URL", "https://djangonaut.space")` to support a staging environment in the future.